### PR TITLE
Template merged into verifyer

### PR DESCRIPTION
**Pull request by ZeniaHansen**
## Description

Merged template into verifyer

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Feature that assists further development (non-breaking change for the developer)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?
Mokey test 

### Checklist:

- [X] You have added a clear title and description for the PR
- [X] The CI pass
- [X] There are no merge conflicts
- [X] The code satisfies the [code standards](https://docs.google.com/document/d/1R9n9MeuRwz3WSrnWZUjrbmGeosTlu1Xz8f5F7gu2AJE/edit#heading=h.e9xl4xfoldow) 
- [X] I have added docstrings for my code, where needed
- [X] Code implements intended functionality
- [X] All relevant tests pass (Black, White, Integration, Monkey, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Continous Integration](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml)
```
Comment (frejaz) [inline README.md]:
```suggestion
[![Continous Integration](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_integration.yml)
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Continous Integration](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_integration.yml)
+[![Continous Delivery](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml)
```
Comment (frejaz) [inline README.md]:
```suggestion
[![Continous Delivery](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_delivery.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Verifiers/actions/workflows/continous_delivery.yml)
```

**Review by Brandhoej — APPROVED** [review]
Looks fine

**Review by Dieuer — APPROVED** [review]
Looks good

**Pull request merged**